keras_retinanet/preprocessing/open_images.py

```diff
@@ -1,3 +1,19 @@
+"""
```

Comment:
How is this classed as an addition from the commit? As the current master has this license in place for this exact file?